src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo';
+import { Comments } from '../../types/Comments';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  comments: Comments[];
+}
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <div className="CommentList">
+    {comments.length > 0
```

Comment:
you can make it simple
```suggestion
    {comments.length
```